### ISSUE TITLE
Govcloud: UnsupportedOperation: Specifying Unlimited mode for an instance is not supported in this region.

### ISSUE THREAD

**Issue by jseiser** [terraform-aws-modules/terraform-aws-ec2-instance#45]
Upgraded the module today, and couldnt figure out why it kept trying to replace my existing instance.  After finally relenting and allowing it to replace, it failed to launch with.

```
Error: Error applying plan:

1 error(s) occurred:

* module.mgmt_bastion.aws_instance.this_t2: 1 error(s) occurred:

* aws_instance.this_t2: Error launching source instance: UnsupportedOperation: Specifying Unlimited mode for an instance is not supported in this region.
	status code: 400, request id: 7609c797-a1a3-4b0b-bc28-e003951ea872

```

This feature is not available in govcloud, so the module no longer works in this region. I assume there is no workaround for this, until terraform supports passing in nulls.

**Comment by antonbabenko** [comment]
It seems that it AWS has made it available in GovCloud recently - https://aws.amazon.com/about-aws/whats-new/2018/08/amazon-ec2-t2-unlimited-instances-now-available-in-aws-govcloud-region/

Please confirm that it is working as expected.

PS: It is pretty hard to follow which feature is available in which AWS region, so thanks for heads up!

**Comment by jojohappy** [comment]
I'm using the AWS in China region, it is not available support unlimited mode for T2/3. I meet the same error when I manage the T2/3 instance. So how can I skip the resource variable? it is just for me to use a custom module extend the `terraform-aws-ec2-instance`?

**Comment by antonbabenko** [comment]
If it is not supported in your region then you have to fork this module and disable it.

Closing this issue.
